input/fsh/SD_AdverseEvent.fsh

```diff
@@ -0,0 +1,44 @@
+Profile: CTCAdverseEvent
+Parent: AdverseEvent
+Title: "CTC Adverse Event"
+Id: ctc-adverse-event
+Description: "Profile of adverse event, using common terminology criteria (CTC)."
+* subject only Reference(Patient)
+* actuality = #actual
+* event 1..1
+* event from MedDRAVS (required)
```

Comment:
Make this (preferrred) since it's possible to send NCIt for CTCAE grade for NCI reporting.